plugins/source/oracledb/client/list_tables.go

```diff
@@ -57,10 +57,14 @@ func (c *Client) updateTableConstraints(ctx context.Context, table *schema.Table
 
 func (c *Client) listTables(ctx context.Context) (schema.Tables, error) {
 	// Please note we don't use ORDER BY here because it's slower than sorting in memory via Go sort.SliceStable
+	// See more about the default namespaces we exclude in https://www.oracletutorial.com/oracle-administration/oracle-tablespace/
 	query := `
 	SELECT ALL_TAB_COLS.TABLE_NAME, COLUMN_ID, COLUMN_NAME, DATA_TYPE, DATA_LENGTH, NULLABLE FROM ALL_TABLES
 	LEFT JOIN ALL_TAB_COLS ON ALL_TABLES.TABLE_NAME = ALL_TAB_COLS.TABLE_NAME
-	WHERE ALL_TABLES.TABLESPACE_NAME = 'USERS'
+	WHERE ALL_TABLES.TABLESPACE_NAME != 'SYSAUX'
+	 AND ALL_TABLES.TABLESPACE_NAME != 'SYSTEM' 
+	 AND ALL_TABLES.TABLESPACE_NAME != 'TEMP' 
+	 AND ALL_TABLES.TABLESPACE_NAME != 'UNDOTBS1'
```

Comment:
I propose to use `NOT IN` instead
```suggestion
	WHERE ALL_TABLES.TABLESPACE_NAME NOT IN ('SYSAUX', 'SYSTEM', 'TEMP', 'UNDOTBS1')
```